### DISCUSSION TITLE
Open the menu with the keyboard

### DISCUSSION THREAD

**Discussion by ivanottantuno** [WayfireWM/wayfire#1103] [Q&A]
Hi,
it's possible to open the menu with the keyboard, i.e. to assign a key binding to the opening of the menu?
Thanks,
Best regards,
Ivan

**Comment by ammen99** [comment]
This is not possible yet (at least with wf-panel), see https://github.com/WayfireWM/wf-shell/issues/11

**Comment by ivanottantuno** [comment]
I understand. But I was thinking, is there the possibility to emulate the
click of the left mouse button on an exact point of the screen? For example
25pixel right and 25 pixel down, from the top left angle?

**Reply by ammen99** [reply]
Sorry, for some reason I missed your question all this time. If you still need the info: it is possible with the `virtual-pointer` protocol. Here's an example how to use it with C code: https://github.com/swaywm/wlroots/blob/master/examples/virtual-pointer.c

Alternatively, you can create a Wayfire plugin, and again create a virtual device and send clicks. See [this plugin](https://github.com/dkondor/wstroke), it is quite advanced but also has an example of creating a virtual device.
